aten/src/ATen/native/take.cpp

```diff
@@ -0,0 +1,105 @@
+#include <ATen/ATen.h>
```

Comment:
There's probably a better name for this file? The file with the legacy definition just lumped together all of the remaining legacy function definitions. Or I could move everything here into TensorAdvancedIndexing.cpp where `take_backward()` lives.